[2025-06-21T07:36:00.949+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-21T07:36:00.968+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment manual__2025-06-21T07:35:59.851764+00:00 [queued]>
[2025-06-21T07:36:00.977+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment manual__2025-06-21T07:35:59.851764+00:00 [queued]>
[2025-06-21T07:36:00.978+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-06-21T07:36:00.992+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): validate_environment> on 2025-06-21 07:35:59.851764+00:00
[2025-06-21T07:36:01.000+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=304) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-21T07:36:01.001+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'validate_environment', 'manual__2025-06-21T07:35:59.851764+00:00', '--job-id', '130', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmpdlqxynlc']
[2025-06-21T07:36:01.002+0000] {standard_task_runner.py:72} INFO - Started process 305 to run task
[2025-06-21T07:36:01.002+0000] {standard_task_runner.py:105} INFO - Job 130: Subtask validate_environment
[2025-06-21T07:36:01.050+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.validate_environment manual__2025-06-21T07:35:59.851764+00:00 [running]> on host 8e3e0e7b2f03
[2025-06-21T07:36:01.138+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Jasjot Parmar' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='validate_environment' AIRFLOW_CTX_EXECUTION_DATE='2025-06-21T07:35:59.851764+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-21T07:35:59.851764+00:00'
[2025-06-21T07:36:01.139+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-21T07:36:01.153+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-06-21T07:36:01.155+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', ' \n        echo "Validating environment..."\n        test -f /app/config.yaml &&\n        test -f /app/.env &&\n        echo "Environment is valid!"\n        ']
[2025-06-21T07:36:01.169+0000] {subprocess.py:99} INFO - Output:
[2025-06-21T07:36:01.172+0000] {subprocess.py:106} INFO - Validating environment...
[2025-06-21T07:36:01.172+0000] {subprocess.py:106} INFO - Environment is valid!
[2025-06-21T07:36:01.173+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-06-21T07:36:01.199+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-21T07:36:01.200+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=validate_environment, run_id=manual__2025-06-21T07:35:59.851764+00:00, execution_date=20250621T073559, start_date=20250621T073600, end_date=20250621T073601
[2025-06-21T07:36:01.258+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-21T07:36:01.286+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-21T07:36:01.288+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
